### PR TITLE
feat: update omnect-device-service to 0.15.1

### PR DESCRIPTION
**Pull request by empwilli**

### CODE REVIEW

File: classes/omnect_rust_modemmanager_deps.bbclass [hunk 1]
```diff
@@ -0,0 +1,29 @@
+CARGO_NET_OFFLINE ?= "true"
```
Comment (mlilien) [inline classes/omnect_rust_modemmanager_deps.bbclass:1]: not to change, but marker for me: we now do this in 3 classes -> should be refactored

File: classes/omnect_rust_modemmanager_deps.bbclass [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+verify_frozen() {
+    cd ${S}/${CARGO_WORKSPACE_ROOT}; cargo verify-project --frozen || bberror "Cargo.lock not uptodate"
```
Comment (mlilien) [inline classes/omnect_rust_modemmanager_deps.bbclass:4]: not to change, but marker for me: we now do this in 3 classes -> should be refactored

File: recipes-omnect/omnect-device-service/omnect-device-service.inc [hunk 3]
```diff
@@ -25,19 +25,22 @@ LIC_FILES_CHKSUM = " \
...
     azure-iot-sdk-c \
+    modemmanager \
     pkgconfig-native \
 "
 
 RDEPENDS:${PN} += "\
     aziot-identityd\
     ca-certificates\
+    modemmanager \
+    modemmanager-dev \
```
Comment (mlilien) [inline recipes-omnect/omnect-device-service/omnect-device-service.inc:36]: runtime dependency on a dev package? why?

File: recipes-omnect/omnect-device-service/omnect-device-service.inc [hunk 4]
```diff
@@ -25,19 +25,22 @@ LIC_FILES_CHKSUM = " \
 
 DEPENDS += "\
     azure-iot-sdk-c \
+    modemmanager \
```
Comment (mlilien) [inline recipes-omnect/omnect-device-service/omnect-device-service.inc:28]: should imho be runtime dependency only

Comment (empwilli) [reply recipes-omnect/omnect-device-service/omnect-device-service.inc:28]: ack

File: recipes-omnect/omnect-device-service/omnect-device-service.inc (hunk 3 above)
Comment (empwilli) [reply recipes-omnect/omnect-device-service/omnect-device-service.inc]: it's not necessary, has been dropped.

**Review by mlilien — APPROVED** [review]

**Review by JanZachmann — CHANGES_REQUESTED** [review]
1. please add a link to the omnect-device-service tag in PR description and maybe add some words about the feature
2. let's talk about testing in our dev channel

**Comment by mlilien** [comment]
obsolete by #395

**Pull request closed without merging**